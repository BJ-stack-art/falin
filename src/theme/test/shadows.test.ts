import { describe, expect, test } from "vitest"
import { shadows, shadowsArray } from "../shadows"

describe("Theme Shadows", () => {
  test("should match shadows object variants", () => {
    expect(shadows).toMatchSnapshot()
  })

  test("should match shadows array", () => {
    expect(shadowsArray).toMatchSnapshot()
  })
})
