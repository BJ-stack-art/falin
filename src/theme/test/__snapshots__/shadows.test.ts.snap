// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Theme Shadows > should match scss variables snapshot 1`] = `
{
  "data": [
    36,
    115,
    104,
    97,
    100,
    111,
    119,
    45,
    120,
    115,
    58,
    32,
    48,
    112,
    120,
    32,
    49,
    112,
    120,
    32,
    50,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    48,
    53,
    41,
    59,
    13,
    10,
    36,
    115,
    104,
    97,
    100,
    111,
    119,
    45,
    115,
    109,
    58,
    32,
    48,
    112,
    120,
    32,
    49,
    112,
    120,
    32,
    51,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    49,
    41,
    44,
    13,
    10,
    32,
    32,
    48,
    112,
    120,
    32,
    49,
    112,
    120,
    32,
    50,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    48,
    54,
    41,
    59,
    13,
    10,
    36,
    115,
    104,
    97,
    100,
    111,
    119,
    45,
    109,
    100,
    58,
    32,
    48,
    112,
    120,
    32,
    52,
    112,
    120,
    32,
    56,
    112,
    120,
    32,
    45,
    50,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    49,
    41,
    44,
    13,
    10,
    32,
    32,
    48,
    112,
    120,
    32,
    50,
    112,
    120,
    32,
    52,
    112,
    120,
    32,
    45,
    50,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    48,
    54,
    41,
    59,
    13,
    10,
    36,
    115,
    104,
    97,
    100,
    111,
    119,
    45,
    108,
    103,
    58,
    32,
    48,
    112,
    120,
    32,
    49,
    50,
    112,
    120,
    32,
    49,
    54,
    112,
    120,
    32,
    45,
    52,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    48,
    56,
    41,
    44,
    13,
    10,
    32,
    32,
    48,
    112,
    120,
    32,
    52,
    112,
    120,
    32,
    54,
    112,
    120,
    32,
    45,
    50,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    48,
    51,
    41,
    59,
    13,
    10,
    36,
    115,
    104,
    97,
    100,
    111,
    119,
    45,
    120,
    108,
    58,
    32,
    48,
    112,
    120,
    32,
    50,
    48,
    112,
    120,
    32,
    50,
    52,
    112,
    120,
    32,
    45,
    52,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    48,
    56,
    41,
    44,
    13,
    10,
    32,
    32,
    48,
    112,
    120,
    32,
    56,
    112,
    120,
    32,
    56,
    112,
    120,
    32,
    45,
    52,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    48,
    51,
    41,
    59,
    13,
    10,
    36,
    115,
    104,
    97,
    100,
    111,
    119,
    45,
    50,
    120,
    108,
    58,
    32,
    48,
    112,
    120,
    32,
    50,
    52,
    112,
    120,
    32,
    52,
    56,
    112,
    120,
    32,
    45,
    49,
    50,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    49,
    56,
    41,
    59,
    13,
    10,
    36,
    115,
    104,
    97,
    100,
    111,
    119,
    45,
    51,
    120,
    108,
    58,
    32,
    48,
    112,
    120,
    32,
    51,
    50,
    112,
    120,
    32,
    54,
    52,
    112,
    120,
    32,
    45,
    49,
    50,
    112,
    120,
    32,
    114,
    103,
    98,
    97,
    40,
    49,
    54,
    44,
    32,
    50,
    52,
    44,
    32,
    52,
    48,
    44,
    32,
    48,
    46,
    49,
    52,
    41,
    59,
    13,
    10,
  ],
  "type": "Buffer",
}
`;

exports[`Theme Shadows > should match shadows array 1`] = `
[
  "none",
  "0px 1px 2px rgba(16, 24, 40, 0.05)",
  "0px 1px 3px rgba(16, 24, 40, 0.1), 0px 1px 2px rgba(16, 24, 40, 0.06)",
  "0px 4px 8px -2px rgba(16, 24, 40, 0.1), 0px 2px 4px -2px rgba(16, 24, 40, 0.06)",
  "0px 12px 16px -4px rgba(16, 24, 40, 0.08), 0px 4px 6px -2px rgba(16, 24, 40, 0.03)",
  "0px 20px 24px -4px rgba(16, 24, 40, 0.08), 0px 8px 8px -4px rgba(16, 24, 40, 0.03)",
  "0px 24px 48px -12px rgba(16, 24, 40, 0.18)",
  "0px 32px 64px -12px rgba(16, 24, 40, 0.14)",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
  "none",
]
`;

exports[`Theme Shadows > should match shadows object variants 1`] = `
{
  "2xl": "0px 24px 48px -12px rgba(16, 24, 40, 0.18)",
  "3xl": "0px 32px 64px -12px rgba(16, 24, 40, 0.14)",
  "lg": "0px 12px 16px -4px rgba(16, 24, 40, 0.08), 0px 4px 6px -2px rgba(16, 24, 40, 0.03)",
  "md": "0px 4px 8px -2px rgba(16, 24, 40, 0.1), 0px 2px 4px -2px rgba(16, 24, 40, 0.06)",
  "sm": "0px 1px 3px rgba(16, 24, 40, 0.1), 0px 1px 2px rgba(16, 24, 40, 0.06)",
  "xl": "0px 20px 24px -4px rgba(16, 24, 40, 0.08), 0px 8px 8px -4px rgba(16, 24, 40, 0.03)",
  "xs": "0px 1px 2px rgba(16, 24, 40, 0.05)",
}
`;
